src/Sarif.Viewer.VisualStudio/ErrorListService.cs

```diff
@@ -175,9 +191,12 @@ private void WriteRunLogToErrorList(RunLog runLog)
                         SarifError sarifError = new SarifError(document)
                         {
                             Region = region,
-                            ErrorCode = result.RuleId,
+                            RuleId = result.RuleId,
+                            RuleName = rule?.Name,
                             Kind = ResultKind.Note,
-                            Message = message,
+                            Category = "Related Location", // or should we prefer original result category?
```

Comment:
Yes. That's why I suggest we move related locations to the expanded area and remove them as discrete entries. I am going to use some ModernCop SARIF data to drive this. You are welcome to implement!! :) We'll need clickable links in the text box.